code/modules/species/species.dm

```diff
@@ -458,6 +457,10 @@ var/global/const/DEFAULT_SPECIES_HEALTH = 200
 
 //Checks if an existing limbs is the specie's default
 /decl/species/proc/is_default_limb(var/obj/item/organ/external/E)
+	if((species_flags & SPECIES_FLAG_CRYSTALLINE) != BP_IS_CRYSTAL(E))
+		return FALSE
+	if((species_flags & SPECIES_FLAG_SYNTHETIC) != BP_IS_PROSTHETIC(E))
```

Comment:
You can't check for equality safely like this; they are bitflags and might differ. Add `!`s or use logic operators.